steely/plugins/flirty.py

```diff
@@ -0,0 +1,51 @@
+'''Dirty talk like you're in Dundalk'''
+
+import random
+import re
+import string
+
+__author__ = ('iandioch')
+COMMAND = '.flirt'
```

Comment:
dot in command name 💯

Reply:
technically i spotted this in #140 so i also deserve credit for this thank ya sir